wachter/bot.py

```diff
@@ -29,7 +30,7 @@
     }
 }
 
-logging.config.dictConfig(log_config)
+config.dictConfig(log_config)
```

Comment:
unrelated but easier to do here, fixes earlier commit which I was unable to test